pages/common/bedops.md

```diff
@@ -0,0 +1,24 @@
+# bedops
+
+> BEDOPS: A fast, highly scalable, and easily-parallelizable genome analysis toolkit.
```

Comment:
The user already knows the name of the command. So I think we can remove "BEDOPS:".

Reply:
BEDOPS is the name of the toolkit. The `bedops` binary is one tool in this kit. There are other binaries that are part of BEDOPS.

Reply:
Ah I see. Ok we can keep it in that case.